### ISSUE TITLE
[qmake] Is building with qmake possible?

### ISSUE THREAD

**Issue by a-andreyev** [hughsie/PackageKit-Qt#32]
Hello!
Is building with qmake possible?

I'm taking [packagekit-qt5](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/packagekit-qt5) package from manjaro/archlinux and trying to use it with cmake projects like [kde's apper](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=apper-git) -- successfully.

And then trying qmake-based project like with:

```
CONFIG += c++11 hide_symbols link_pkgconfig
PKGCONFIG += ssu-sysinfo nemodbus packagekitqt5
...
```

and getting compilation error about missing headers:
```
#include <packagekitqt_global.h>
```

The workaround is probably to change includes to `"packagekitqt_global.h"` but I'm not a pro with build systems.
Any hints to check twice it's not my system env/compile flag problems?

**Comment by a-andreyev** [comment]
could be related to #26 somehow :thinking:

**Comment by ximion** [comment]
Yes, that pkg-config bug looks very much related. Do you have QPK 1.0.2 installed, which includes a fix for this issue?
